selectors/rhs.jsx

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2017 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {createSelector} from 'reselect';
+import {localizeMessage} from 'utils/utils.jsx';
+import {PostTypes} from 'utils/constants.jsx';
+
+export function getSelectedPostId(state) {
+    return state.views.rhs.selectedPostId;
+}
+
+function getSelectedPostChannelId(state) {
+    return state.views.rhs.selectedPostChannelId;
+}
+
+function getCurrentUserId(state) {
```

Comment:
This can be imported from `mattermost-redux/selectors/entities/users`

Reply:
Done.